Tasks/DotNetCoreCLI/dotnetcore.ts

```diff
@@ -58,6 +62,13 @@ export class dotNetExe {
         }
     }
 
+    private  setConsoleCodePage() {
+        // set the console code page to "UTF-8"
+        if (tl.osType() === 'Windows_NT') {
```

Comment:
Also, does it need to be done in other tasks where we are invoking exes like az cli, packer

Reply:
Any exception thrown from this should be ignored with a warning